frontend/www/js/omegaup/components/arena/ContestListv2.vue

```diff
@@ -9,37 +9,40 @@
         pills
         card
         vertical
-        nav-wrapper-class="contest-list-nav col-sm-4 col-md-2"
+        nav-wrapper-class="contest-list-nav  col-md-2 col-sm-12 test-class"
       >
         <b-card>
           <b-container>
             <b-row class="p-1" align-v="center">
-              <b-col cols="6">
+              <b-col class="col-md-6 col-sm-12">
                 <form :action="queryURL" method="GET">
-                  <div class="input-group">
-                    <input
-                      v-model.lazy="currentQuery"
-                      class="form-control"
-                      type="text"
-                      name="query"
-                      autocomplete="off"
-                      autocorrect="off"
-                      autocapitalize="off"
-                      spellcheck="false"
-                      :placeholder="T.wordsKeyword"
-                    />
-                    <button class="btn" type="reset">&times;</button>
-                    <div class="input-group-append">
-                      <input
-                        class="btn btn-primary btn-md active"
-                        type="submit"
-                        :value="T.wordsSearch"
-                      />
+                  <div class="row">
+                    <div class="col-12 col-sm-8">
+                      <div class="input-group-append">
```

Comment:
Acá este div es el que agrupa los elementos, entonces la clase debería ser
```suggestion
                      <div class="input-group">
```